addons/mail/static/src/models/media_preview/media_preview.js

```diff
@@ -0,0 +1,173 @@
+/** @odoo-module **/
+
+import { attr } from '@mail/model/model_field';
+import { registerNewModel } from '@mail/model/model_core';
+
+function factory(dependencies) {
+
+    class MediaPreview extends dependencies['mail.model'] {
+
+        //----------------------------------------------------------------------
+        // Public
+        //----------------------------------------------------------------------
+
+        /**
+         * Iterates tracks of the provided MediaStream, calling the `stop`
+         * method on each of them.
+         * 
+         * @static
+         * @param {MediaStream} mediaStream 
+         */
+        static stopTracksOnMediaStream(mediaStream) {
```

Comment:
Why a static method with a param? Just make it a normal method and not param instead?